torch/ao/quantization/fx/match_utils.py

```diff
@@ -12,7 +12,8 @@
     QConfigAny,
 )
 from ..utils import (
-    MatchAllNode
+    MatchAllNode,
+    conv_add_pattern_list
```

Comment:
I feel we probably don't want to hack for special patterns in the general match_utils.py function..

Reply:
Thanks for the comment. Per your reply here https://github.com/pytorch/pytorch/issues/96288#issuecomment-1487488569, should we wait for the new programmable API ready to fix this issue?

Reply:
yes, it should be ready in 2 weeks

Reply:
That's great. Will check the fix after it's ready. Thanks~